code/midtown/mmwidget/menu.cpp

```diff
@@ -74,6 +74,11 @@ i32 UIMenu::IsAnOptionMenu()
     return 0;
 }
 
+void UIMenu::AssignBackground(char* background_name)
```

Comment:
When dealing with `char*` parameters, you'll usually want to change their types to `aconst char*` (unless their contents are actually supposed to be modified).

The `aconst` is to allow stricter type checking when compiling with `ARTS_STANDALONE`. Try temporarily commenting out the define at the top of arts_pch.h - it will fail with a bunch of linker errors, but shouldn't give actual compilation errors.